src/compiler/commandLineParser.ts

```diff
@@ -536,21 +589,24 @@ namespace ts {
       *    file to. e.g. outDir
       */
     export function parseJsonConfigFileContent(json: any, host: ParseConfigHost, basePath: string, existingOptions: CompilerOptions = {}, configFileName?: string): ParsedCommandLine {

```

Comment:
Yea. it can be. Will be better to have it as optional as well. I also realize that `convertOptionFromJson` doesn't use the parameter at all